tests/test_sims_corr.py

```diff
@@ -0,0 +1,73 @@
+import unittest
```

Comment:
you will also need tests once you write input checks

Reply:
tests that make sure that wrong values of the parameters are not allowed by the function. See examples in the simulations tests of how to do this